src/metax_api/api/rest/base/views/file_view.py

```diff
@@ -149,6 +149,10 @@ def datasets(self, request):
         parameters in GET. Also, some clients forcibly shove parameters in body in GET
         requests to query parameters, so using POST instead is more guaranteed to work.
         """
+
+        if request.query_params and request.query_params['detailed'] == 'true':
```

Comment:
Maybe add the use of CommonService.get_boolean_query_param(request, 'detailed') which allows the use of parameter as "?detailed" which returns true...I understood this has done with other parameters as well.